src/components/PageMetadata.tsx

```diff
@@ -21,230 +21,86 @@ type PropMeta = {
-        ogImageDevelopers: file(relativePath: { eq: "enterprise-eth.png" }) {
-          childImageSharp {
-            gatsbyImageData(
-              width: 1200
-              layout: FIXED
-              placeholder: BLURRED
-              quality: 100
-            )
-          }
-        }
-        ogImageDapps: file(relativePath: { eq: "doge-computer.png" }) {
-          childImageSharp {
-            gatsbyImageData(
-              width: 1200
-              layout: FIXED
-              placeholder: BLURRED
-              quality: 100
-            )
-          }
-        }
-        ogImageUpgrades: file(
-          relativePath: { eq: "upgrades/upgrade_doge.png" }
-        ) {
-          childImageSharp {
-            gatsbyImageData(
-              width: 1200
-              layout: FIXED
-              placeholder: BLURRED
-              quality: 100
-            )
-          }
-        }
-      }
-    `
-  )
+  const { locale, asPath } = useRouter()
 
-  const location = useLocation()
+  // TODO: Re-implement intl
+  // const { t } = useTranslation()
 
-  const { t } = useTranslation()
-  const { language } = useI18next()
+  // const desc = description || t("site-description")
+  const desc =
+    description ||
+    "Ethereum is a global, decentralized platform for money and new kinds of applications. On Ethereum, you can write code that controls money, and build applications accessible anywhere in the world."
 
-  const desc = description || t("site-description")
-
-  const siteTitle = t("site-title")
+  // const siteTitle = t("site-title")
+  const siteTitle = "ethereum.org"
+  const fullTitle = `${title} | ${siteTitle}`
 
   /* Set canonical URL w/ language path to avoid duplicate content */
   /* e.g. set ethereum.org/about/ to ethereum.org/en/about/ */
-  const { pathname } = location
-  let canonicalPath = pathname
-  const firstDirectory = canonicalPath.split("/")[1]
-  if (!isLang(firstDirectory)) {
-    canonicalPath = `/en${pathname}`
-  }
-  const canonical = canonicalUrl || `${site.siteMetadata.url}${canonicalPath}`
+  const canonical = canonicalUrl || join(SITE_URL, DEFAULT_LOCALE, asPath)
+  const url = locale ? join(SITE_URL, locale, asPath) : canonical
 
   /* Set fallback ogImage based on path */
-  const siteUrl = site.siteMetadata.url
-  let ogImage = getSrc(ogImageDefault)
-  if (pathname.includes("/developers/")) {
-    ogImage = getSrc(ogImageDevelopers)
+  let ogImage = ogImageDefault.src
+  if (asPath.includes("/developers/")) {
+    ogImage = ogImageDevelopers.src
   }
-  if (pathname.includes("/dapps/")) {
-    ogImage = getSrc(ogImageDapps)
+  if (asPath.includes("/dapps/")) {
+    ogImage = ogImageDapps.src
   }
-  if (pathname.includes("/roadmap/")) {
-    ogImage = getSrc(ogImageUpgrades)
+  if (asPath.includes("/roadmap/")) {
+    ogImage = ogImageUpgrades.src
   }
   if (image) {
     ogImage = image
   }
-  const ogImageUrl = `${siteUrl}${ogImage}`
+  const ogImageUrl = join(SITE_URL, ogImage)
```

Comment:
> do we have a fallback image if frontmatter.image is undefined?

@nhsz To your comment about a fallback, yes... this is where that is handled. If `image` is undefined, the `if` statement on L68 inside `PageMetadata` will not execute, and it will fallback to whatever one has been determined based on path above.